certbot_dns_loopia.py

```diff
@@ -26,15 +28,15 @@ class LoopiaAuthenticator(DNSAuthenticator):
     description = __doc__.strip().split("\n", 1)[0]
 
     #: TTL for the validation TXT record
-    ttl = 30
+    ttl = DEFAULT_TTL
```

Comment:
I think using `DEFAULT_TTL` here implies that it's possible to configure this. Since it isn't I'd prefer to keep the `30` here.

If you want the test case to be a bit clearer you can use `LoopiaAuthenticator.ttl`